client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -89,9 +89,7 @@ def log_model(
                 - ``str`` path to a file or directory
                 - arbitrary ``pickle``\ able object
         custom_modules : list of str, optional
-            Paths to local Python modules and other files that the deployed
-            model depends on. Modules from the standard library should not be
-            included here.
+            Paths to local Python modules and other files that the deployed model depends on. Modules from the standard library should not be included here.
```

Comment:
The 3 lines was causing a build warning in Sphinx 🤷🏼‍♂️ so I'm combining them into one line.